app/AppKernel.php

```diff
@@ -54,6 +54,7 @@ public function registerBundles()
 
             new FOS\ElasticaBundle\FOSElasticaBundle(),
             new SWP\Bundle\ElasticSearchBundle\SWPElasticSearchBundle(),
+            new OldSound\RabbitMqBundle\OldSoundRabbitMqBundle(),
```

Comment:
Do we use it somewhere? I don't see any usage of this besides the configuration.

Reply:
we will in next PR, don't want to remove it now.